[2024-05-05 14:14:22] EyeGazeServer INFO: Application start
[2024-05-05 14:14:27] EyeGazeServer INFO: Client ('127.0.0.1', 50326) connected. Total clients: 0
[2024-05-05 14:14:33] EyeGazeServer ERROR: Socket error: [WinError 10054] Удаленный хост принудительно разорвал существующее подключение
Traceback (most recent call last):
  File "c:\Users\bokar\Documents\EyeGazeFrameworkDemo\server.py", line 170, in client_handler
    if not self.process_frame(conn):
  File "c:\Users\bokar\Documents\EyeGazeFrameworkDemo\server.py", line 134, in process_frame
    self.handle_image_processing_and_sending(img, conn)
  File "c:\Users\bokar\Documents\EyeGazeFrameworkDemo\server.py", line 148, in handle_image_processing_and_sending
    self.send_data(conn, data)
  File "c:\Users\bokar\Documents\EyeGazeFrameworkDemo\server.py", line 128, in send_data
    conn.sendall(message_length.to_bytes(4, 'big'))
ConnectionResetError: [WinError 10054] Удаленный хост принудительно разорвал существующее подключение

[2024-05-05 14:14:33] EyeGazeServer INFO: Client ('127.0.0.1', 50326) disconnected. Total clients: 0
[2024-05-05 15:17:19] EyeGazeServer INFO: Application start
[2024-05-05 15:17:27] EyeGazeServer INFO: Client ('127.0.0.1', 51875) connected. Total clients: 0
[2024-05-05 15:18:05] EyeGazeServer ERROR: Socket error: [WinError 10054] Удаленный хост принудительно разорвал существующее подключение
Traceback (most recent call last):
  File "c:\Users\bokar\Documents\EyeGazeFrameworkDemo\server.py", line 172, in client_handler
    if not self.process_frame(conn):
  File "c:\Users\bokar\Documents\EyeGazeFrameworkDemo\server.py", line 136, in process_frame
    self.handle_image_processing_and_sending(img, conn)
  File "c:\Users\bokar\Documents\EyeGazeFrameworkDemo\server.py", line 150, in handle_image_processing_and_sending
    self.send_data(conn, data)
  File "c:\Users\bokar\Documents\EyeGazeFrameworkDemo\server.py", line 130, in send_data
    conn.sendall(message_length.to_bytes(4, 'big'))
ConnectionResetError: [WinError 10054] Удаленный хост принудительно разорвал существующее подключение

[2024-05-05 15:18:05] EyeGazeServer INFO: Client ('127.0.0.1', 51875) disconnected. Total clients: 0
[2024-05-05 15:39:10] EyeGazeServer INFO: Application start
[2024-05-05 15:39:49] EyeGazeServer INFO: Application start
[2024-05-05 15:40:02] EyeGazeServer INFO: Client ('127.0.0.1', 52481) connected. Total clients: 0
[2024-05-05 15:40:11] EyeGazeServer ERROR: Socket error: [WinError 10054] Удаленный хост принудительно разорвал существующее подключение
Traceback (most recent call last):
  File "c:\Users\bokar\Documents\EyeGazeFrameworkDemo\server.py", line 171, in client_handler
    if not self.process_frame(conn):
  File "c:\Users\bokar\Documents\EyeGazeFrameworkDemo\server.py", line 135, in process_frame
    self.handle_image_processing_and_sending(img, conn)
  File "c:\Users\bokar\Documents\EyeGazeFrameworkDemo\server.py", line 149, in handle_image_processing_and_sending
    self.send_data(conn, data)
  File "c:\Users\bokar\Documents\EyeGazeFrameworkDemo\server.py", line 129, in send_data
    conn.sendall(message_length.to_bytes(4, 'big'))
ConnectionResetError: [WinError 10054] Удаленный хост принудительно разорвал существующее подключение

[2024-05-05 15:40:11] EyeGazeServer INFO: Client ('127.0.0.1', 52481) disconnected. Total clients: 0
[2024-05-05 19:12:38] EyeGazeServer INFO: Application start
[2024-05-05 19:12:38] EyeGazeServer INFO: Server listening on 127.0.0.1:9556
[2024-05-05 19:12:51] EyeGazeServer INFO: Client ('127.0.0.1', 58367) connected. Total clients: 0
[2024-05-05 19:13:13] EyeGazeServer ERROR: Socket error: [WinError 10054] Удаленный хост принудительно разорвал существующее подключение
Traceback (most recent call last):
  File "c:\Users\bokar\Documents\EyeGazeFrameworkDemo\server.py", line 173, in client_handler
    if not self.process_frame(conn):
  File "c:\Users\bokar\Documents\EyeGazeFrameworkDemo\server.py", line 137, in process_frame
    self.handle_image_processing_and_sending(img, conn)
  File "c:\Users\bokar\Documents\EyeGazeFrameworkDemo\server.py", line 151, in handle_image_processing_and_sending
    self.send_data(conn, data)
  File "c:\Users\bokar\Documents\EyeGazeFrameworkDemo\server.py", line 131, in send_data
    conn.sendall(message_length.to_bytes(4, 'big'))
ConnectionResetError: [WinError 10054] Удаленный хост принудительно разорвал существующее подключение

[2024-05-05 19:13:13] EyeGazeServer INFO: Client ('127.0.0.1', 58367) disconnected. Total clients: 0
[2024-05-05 19:13:23] EyeGazeServer INFO: No active clients. Shutting down the server.
